build.sbt

```diff
@@ -50,6 +55,29 @@ lazy val `arrows-stdlib` =
 lazy val `arrows-stdlib-jvm` = `arrows-stdlib`.jvm
 lazy val `arrows-stdlib-js` = `arrows-stdlib`.js.settings(test := {})
 
+lazy val `arrows-stdlib-cats` =
+  crossProject.crossType(superPure)
+  .settings(commonSettings)
+  .settings(
+    crossScalaVersions := Seq("2.12.5"),
+    name := "arrows-stdlib-cats",
+    libraryDependencies ++= List(
+      "org.typelevel" %%% "cats-effect" % "1.0.0-RC3",
+      "org.typelevel" %%% "cats-effect-laws" % "1.0.0-RC3",
+      "org.typelevel" %%% "cats-mtl-core" % "0.2.3",
+      "org.typelevel" %%% "cats-mtl-laws" % "0.2.3",
```

Comment:
Here too, I think we only need `cats-mtl-laws` for testing purposes, no reason to depend on it in any other context.